etc/neutron/services/f5/f5-openstack-agent.ini

```diff
@@ -171,6 +171,15 @@ f5_external_physical_mappings = default:1.1:True
 #
 f5_vtep_folder = 'Common'
 f5_vtep_selfip_name = 'vtep'
+
+######################################################################
+#  The following 5 optiosn are necessary for l2adjacent deployments. #
+######################################################################
+#f5_device_type = external
+#f5_vtep_selfip_name = 'selfip.client'
+#advertised_tunnel_types = 'vxlan'
+#f5_global_routed_mode = False 
+#icontrol_hostname = 10.190.7.130
```

Comment:
Revert changes to this file.  We'll create an ini-examples folder.  It's ok to have a specific ini file for your test env located in the devtools dir.